Makefile

```diff
@@ -230,4 +230,4 @@ fmtcheck:
 
 .PHONY: validate_commits
 validate_commits: $(GOPATH)/bin/git-validation
-	git-validation -q -run short-subject,message_regexp='(Issue #[0-9]+ .*|cut v[0-9]+.[0-9]+.[0-9]+)' -range $(START_COMMIT_MESSAGE_VALIDATION)...
+	git-validation -q -run short-subject,message_regexp='(^Issue #[0-9]+ .*|^cut v[0-9]+.[0-9]+.[0-9]+)' -range $(START_COMMIT_MESSAGE_VALIDATION)...
```

Comment:
Shouldn't `^(Issue #[0-9]+ .*|cut v[0-9]+.[0-9]+.[0-9]+)` be equivalent?

Reply:
https://regex101.com/r/Iv69Jr/1

Reply:
`^(Issue #[0-9]+ .*|cut v[0-9]+.[0-9]+.[0-9]+$)`